evm/chains/tester/__init__.py

```diff
@@ -53,11 +58,27 @@ class EIP150TesterVM(MaintainGasLimitMixin, BaseEIP150VM):
 @reversed_return
 def _generate_vm_configuration(homestead_start_block=None,
                                dao_start_block=None,
-                               eip150_start_block=None):
+                               eip150_start_block=None,
+                               spurious_dragon_block=None):
     # If no explicit configuration has been passed, configure the vm to start
     # with the latest fork rules at block 0
-    if eip150_start_block is None and homestead_start_block is None:
-        yield (0, EIP150TesterVM)
+    no_declared_blocks = (
+        spurious_dragon_block is None and
+        eip150_start_block is None and
+        homestead_start_block is None
+    )
+    if no_declared_blocks:
+        yield (0, SpuriousDragonTesterVM)
```

Comment:
We're going to have to change this every time we implement rules for a new fork, right? Is there any way we could keep a list of existing forks (in order) and always pick the last one here?

Reply:
Yes, but I've been punting on writing a generic algorithm for this.  Soon.